### PR TITLE
Bring Gitpod Integration back

### PR DESCRIPTION
**Pull request by mustard-mh**
Bring Gitpod Integration back

Relates ENT-787

### CODE REVIEW

File: src/vs/platform/environment/node/userDataPath.d.ts [hunk 1]
```diff
@@ -3,7 +3,7 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-import { NativeParsedArgs } from 'vs/platform/environment/common/argv';
+import { NativeParsedArgs } from '../../../platform/environment/common/argv.js';
```
Comment (jeanp413) [inline src/vs/platform/environment/node/userDataPath.d.ts:6]: Do we need this change? seems too work as is in upstream repo?

File: build/gulpfile.vscode.web.js [hunk 2]
```diff
@@ -110,6 +115,8 @@ const vscodeWebEntryPoints = !isAMD() ? [
   buildfile.workerBackgroundTokenization,
   buildfile.keyboardMaps,
   buildfile.workbenchWeb(),
+  // Gitpod Integration
+  buildfile.codeWeb,
```
Comment (jeanp413) [inline build/gulpfile.vscode.web.js:119]: why is this needed isn't it already included in `buildfile.workbenchWeb()` ?

**Review by jeanp413 — APPROVED** [review]

File: build/gulpfile.vscode.web.js (hunk 2 above)
Comment (mustard-mh) [reply build/gulpfile.vscode.web.js:119]: There's no `workbench {}.js` without this line

File: src/vs/platform/environment/node/userDataPath.d.ts (hunk 1 above)
Comment (mustard-mh) [reply src/vs/platform/environment/node/userDataPath.d.ts:6]: Yep, we don't need it, update because I tried to global search `from 'vs` to see what's missing

Comment (mustard-mh) [inline src/vs/platform/environment/node/userDataPath.d.ts:6]:
```suggestion
import { NativeParsedArgs } from 'vs/platform/environment/common/argv';
```
